onos-operator/charts/app-operator/values.yaml

```diff
@@ -17,6 +17,11 @@ image:
   pullPolicy: IfNotPresent
   pullSecrets: []
 
+proxyimage:
+  registry: ""
+  repository: onosproject/onos-proxy
+  tag: v0.1.1
```

Comment:
later on we should change this to 0.1.2. I just merged my PR

Reply:
Yeah, let's just wait for the release of the v0.1.2 and change this here as part of this PR.